README.md

```diff
@@ -1,2 +1,34 @@
-# HelloMicroverse
-This is a hello microverse week one project
+# Hello Microverse
+
+> This is the first project at Microverse first week activity.
+
+## Built With
+
+- HTML 
+- CSS
+
+## Authors
+
+👤 **Aster Alemu**
+
+- GitHub: [aster alemu](https://github.com/aster-alemu)
+- Twitter: [Aster Alemu](https://twitter.com/asteralemu)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/aster-alemu)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Great thanks to one of the Micronat Tarikwa Tesfa she helps me a lot
+
+## 📝 License
+
+This project is not licensed 
```

Comment:
- [ ] **[OPTIONAL]** You can add an `MIT` license to your project just by creating a `MIT.md` file in the root directory of your project, that is, the same directory as your `README.md` file. You then copy the `MIT.md` file from this [repository](https://github.com/microverseinc/readme-template/blob/master/MIT.md) into your own `MIT.md` file. Remember to personalise it to your `NAME` and `APP TYPE`. Finally, push your code to GitHub and you will have your MIT licence working 💯 👍